src/DataLayer/converters/tests/MeasureConverter.spec.ts

```diff
@@ -15,36 +14,31 @@ function addFormat(measure: IMeasure, format: string = "$#,##0 custom") {
-        ${"fact-based measure in %, adding default format"}       | ${measures.factBasedMeasureInPercent}                    | ${afm.factBasedMeasureInPercent}
-        ${"fact-based measure in %, keeping custom format"}       | ${addFormat(measures.factBasedMeasureInPercent)}         | ${afm.factBasedMeasureInPercentWithCustomFormat}
-        ${"fact-based measure in %, overriding incorrect format"} | ${addDecFormat(measures.factBasedMeasureInPercent)}      | ${afm.factBasedMeasureInPercent}
-        ${"attribute-based measure, adding default format"}       | ${measures.attributeBasedMeasure}                        | ${afm.attributeBasedMeasure}
-        ${"attribute-based measure, keeping custom format"}       | ${addFormat(measures.attributeBasedMeasure)}             | ${afm.attributeBasedMeasureWithCustomFormat}
-        ${"attribute-based measure in %, adding default format"}  | ${measures.attributeBasedMeasureInPercent}               | ${afm.attributeBasedMeasureInPercent}
-        ${"attribute-based measure in %, keeping custom format"}  | ${addFormat(measures.attributeBasedMeasureInPercent)}    | ${afm.attributeBasedMeasureInPercentWithCustomFormat}
-        ${"attribute-based m. in %, overriding default format"}   | ${addIntFormat(measures.attributeBasedMeasureInPercent)} | ${afm.attributeBasedMeasureInPercent}
-        ${"POP measure, not adding default format"}               | ${measures.popMeasure}                                   | ${afm.popMeasure}
-        ${"POP measure, keeping default format"}                  | ${addFormat(measures.popMeasure)}                        | ${afm.popMeasureWithCustomFormat}
-        ${"previous period measure, not adding default format"}   | ${measures.previousPeriodMeasure}                        | ${afm.previousPeriodMeasure}
-        ${"measure with show in percent, adding default format"}  | ${measures.showInPercentMeasure}                         | ${afm.showInPercentMeasure}
-        ${"measure with show in percent, keeping custom format"}  | ${addFormat(measures.showInPercentMeasure)}              | ${afm.showInPercentWithCustomFormat}
-        ${"arithmetic measure, not adding default format"}        | ${measures.arithmeticMeasure}                            | ${afm.arithmeticMeasure}
-        ${"arithmetic measure, keeping custom format"}            | ${addFormat(measures.arithmeticMeasure)}                 | ${afm.arithmeticMeasureWithCustomFormat}
-        ${"arithmetic measure-change, adding default format"}     | ${measures.arithmeticMeasureChange}                      | ${afm.arithmeticMeasureChange}
-        ${"arithmetic measure-change, keeping custom format"}     | ${addFormat(measures.arithmeticMeasureChange)}           | ${afm.arithmeticMeasureWithChangeOperatorAndCustomFormat}
+        testCase                                                 | inputVizObjMeasure                                    | outputAfmMeasure
```

Comment:
only two lines related to the override have been removed. Prettier probably re-aligned the rest